pkg/generate/app/cmd/template.go

```diff
@@ -53,7 +53,10 @@ func formatString(out io.Writer, tab, s string) {
 	labelVals := strings.Split(s, "\n")
 
 	for _, lval := range labelVals {
-		fmt.Fprintf(out, fmt.Sprintf("%s%s\n", tab, lval))
+		s := fmt.Sprintf("%s%s\n", tab, lval)
+		if len(strings.TrimSpace(s)) > 0 {
+			fmt.Fprintf(out, fmt.Sprintf("%s%s\n", tab, lval))
+		}
```

Comment:
@gabemontero 
> Far as I can tell from the comments in those 2 items, there was no explicit intent to strip out newlines from things like "foo\n\nbar\n" ... does that sound right to you @juanvallejo ?

Correct, I believe the original approach for `formatString` only focused on making sure that all new lines in a given string had the same level of indentation